content/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners.md

```diff
@@ -91,6 +91,7 @@ Your access token will require the following scope:
 
 - For private repositories, use an access token with the [`repo` scope](/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/#available-scopes).
 - For public repositories, use an access token with the [`public_repo` scope](/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/#available-scopes).
+- For organizations, use an access token with the [`admin:org` scope](/rest/reference/actions#get-a-self-hosted-runner-for-an-organization).
```

Comment:
Just making the link go to the scopes page, rather than link to a single API operation (the scopes page better describes what `admin:org` entails.)

```suggestion
- For organizations, use an access token with the [`admin:org` scope](/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/#available-scopes).
```